tests/TestPartialMatch.cpp

```diff
@@ -693,144 +705,153 @@ TEST_P(TestPartialMatch, databaseLookupWorksCorrectlyForCtxtAndPtxt)
     }
 }
 
-TEST(TestPartialMatch, databaseLookupQueryAPIGeneratesPostFix)
+TEST_P(TestPartialMatch, databaseLookupWorksWithQueryAPI)
```

Comment:
The two tests you have added technically just check that the ctxt lookup agrees with the ptxt lookup. Both could technically be outputting incorrect results and we would never know.

Reply:
yeah I asked about this a couple of weeks ago: this was the only test on the current contains() function, so I asked if it was okay to only add the same test on my new contains function.

Reply:
Oh ok I must've missed that question, I'd suggest you have at least one test that checks for correctness of one of the pipelines. You could test the plaintext one if that's easier just so that we can be confident it is definitely correct and then rely on the existing test for the correctness of the ctxt pipeline.

Reply:
I've added a test here for the ptxt ptxt case https://github.com/helibproject/HElib/blob/14c3b9f9f57865c09ae340b7322bdfceb16caa64/tests/TestPartialMatch.cpp#L622